public/docs/ts/latest/cookbook/dynamic-form.jade

```diff
@@ -32,11 +40,27 @@ include ../_util-fns
    **See the [live example](/resources/live-examples/cb-dynamic-form/ts/plnkr.html)**.
 
 .l-main-section
+<a id="bootstrap"></a>
+:marked
+   ## Bootstrap
+
+   During bootstrap we have to register the new forms module by calling `provideForms()` and pass the result to the provider array. 
+
++makeExample('cb-dynamic-form/ts/app/main.ts','','app/main.ts')   
+
+:marked
+   The old forms API is going through a deprecation phase. During this transition we are supporting both form modules. 
```

Comment:
This "we" is confusing in here because I don't understand if you mean "Angular" or this concrete example.

We need to differentiate that "we" because two lines after, we mention that we are going to disable the old forms and the "we" there means the example and not angular.